src/components/TopBar/TopBar.tsx

```diff
@@ -309,7 +309,7 @@ const TopBar = ({
                                         <Box
                                             component="a"
                                             key={item.name}
-                                            href={item.url}
+                                            href={item.url as string}
```

Comment:
why do you cast? it's [already define as string](https://github.com/gridsuite/commons-ui/blob/93c08e7049c05ffb7ce24529873f27b6375c5207/src/hooks/predefined-properties-hook.ts#L21C5-L21C17) in interface

Reply:
Since Url is either` string | URL` and href only accepts string don't we have to explicitely cas url into string ? 
If I don't I've got the following error : 
```
Type Url is not assignable to type string | undefined
Type URL is not assignable to type string
```

Reply:
oh okay, in that case
```suggestion
                                            href={item.url?.toString()}
```

Reply:
Done